packages/core/src/components/cart/CartSidebar/cart-sidebar.module.scss

```diff
@@ -60,6 +44,8 @@
   }
 
   [data-fs-cart-sidebar-footer] {
+    position: sticky;
+    bottom: 0;
```

Comment:
hmm, I wonder if this can make the footer is hidden in a real device. 👀

Reply:
I'll test it again when available on the starter. 🙇